lib/scanner/index.js

```diff
@@ -43,9 +57,44 @@ class Scanner extends Emitter {
   stop() {
     debug('stop');
     MagnetScanner.stop();
+    this.stopExpireCheck();
     this.listener.remove();
   }
 
+  startExpireCheck() {
+    this.nextExpireCheck = setTimeout(() => {
```

Comment:
Humm, I'm starting to think that you leaved like that to be able to enable/disable without having to do a `startExpireCheck` again.